schedule/yast/textmode/textmode.yaml

```diff
@@ -0,0 +1,55 @@
+---
+name: textmode
+description: |
+  Installation in textmode and selecting the textmode "desktop" during installation.
```

Comment:
s/and selecting the textmode "desktop" during installation/which selects System Role 'Text Mode'/ and same for the other files.